src/createAnimatedComponent/NativeEventsManager.ts

```diff
@@ -0,0 +1,144 @@
+'use strict';
+import type {
+  INativeEventsManager,
+  IAnimatedComponentInternal,
+  AnimatedComponentProps,
+  InitialComponentProps,
+  AnimatedComponentRef,
+} from './commonTypes';
+import { has } from './utils';
+import { WorkletEventHandler } from '../reanimated2/WorkletEventHandler';
+import { findNodeHandle } from 'react-native';
+
+export class NativeEventsManager implements INativeEventsManager {
+  _managedComponent: ManagedAnimatedComponent;
+  _componentOptions?: ComponentOptions;
+  _eventViewTag = -1;
+
+  constructor(component: ManagedAnimatedComponent, options?: ComponentOptions) {
+    this._managedComponent = component;
+    this._componentOptions = options;
+    this._eventViewTag = getEventViewTag(
+      this._managedComponent,
+      this._componentOptions
+    );
+  }
+
+  public attachNativeEvents() {
```

Comment:
```suggestion
  public attachEvents() {
```
The class name already includes the `NativeEvents` prefix, so there's no need to repeat this information in the method name. You can apply this change also to `detachNativeEvents` and `updateNativeEvents` methods.

Reply:
smart